sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ProjectionOverSchema.scala

```diff
@@ -26,7 +25,7 @@ import org.apache.spark.sql.types._
  * are adjusted to fit the schema. All other expressions are left as-is. This
  * class is motivated by columnar nested schema pruning.
  */
-private[execution] case class ProjectionOverSchema(schema: StructType) {
+private[sql] case class ProjectionOverSchema(schema: StructType) {
```

Comment:
Since `catalyst` is already an internal module,  `private[sql]` would better be removed.  These were removed out as of [SPARK-16813](https://issues.apache.org/jira/browse/SPARK-16813).